Description: disable ProPolice SSP, which may be enabled by default
Author: Thorsten Glaser

--- a/Makefile	Fri Sep 16 18:17:17 2011
+++ b/Makefile	Fri Sep 16 18:18:14 2011
@@ -141,6 +141,7 @@ CFLAGS = -g
 COMMENT = :
 endif
 CFLAGS += -W -Wall -Wextra -Wchar-subscripts -Wmissing-prototypes -Wmissing-declarations -Wno-switch -Wno-unused -Wredundant-decls
+CFLAGS += -fno-stack-protector
 
 PWD=$(shell pwd)
 
